src/sipp.cpp

```diff
@@ -767,17 +775,33 @@ static pid_t begin_pager() {
         return 0;
     }
 
-    /* Get pager first, so we can bail if it's not there */
-    argv[0] = getenv("PAGER");
-    if (!argv[0]) {
-        argv[0] = pager; /* missing PAGER */
-    } else if (!*argv[0]) {
-        return 0; /* blank PAGER */
+    /* Get pager first, so we can bail if it's not there. Prefer env */
+    if ((ptr = getenv("PAGER"))) {
+        if (!ptr[0]) {
+            return 0; /* blank PAGER */
```

Comment:
!ptr

Reply:
No. `!ptr[0]` or `ptr[0] == '\0'`

The `if` at 779 already checks `!ptr`.